docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -167,7 +167,7 @@ If you log in with Bitbucket, Codacy requires the following [permissions/scopes]
     </tr>
     <tr>
       <td><code>webhook</code></td>
-      <td>Codacy creates webhooks for code pushes and pull request events (created, merged, etc.). These events might trigger code analysis. Codacy also creates repository webhooks to enable <a href="../../organizations/what-are-synced-organizations/">synced organizations</a>.</td>
+      <td>Codacy creates webhooks for code pushes and pull request events (created, merged, etc.). These events might trigger code analysis. Codacy also creates repository webhooks to enable <a href="../../organizations/what-are-organizations/">organizations</a>.</td>
```

Comment:
We still have a "to enable organizations" bit here.

```suggestion
      <td>Codacy creates webhooks for code pushes and pull request events (created, merged, etc.). These events might trigger code analysis. Codacy also creates repository webhooks for <a href="../../organizations/what-are-organizations/">organizations</a>.</td>
```

@prcr Does this change make sense?

Reply:
Yes, let's make sure that we remove all instances of the "to enable organizations ..." from this page. :rocket:

Reply:
```suggestion
      <td>Codacy creates webhooks for code pushes and pull request events (created, merged, etc.). These events might trigger code analysis.</td>
```

@prcr Or should we drop the sentence altogether?